README.md

```diff
@@ -6,9 +6,19 @@
 
 <https://warsawjs.github.io/meetup-slides-template>
 
+## Updating slides to your needs
+
+* rename project at `GitHub` with snake-case convention
+* `description` and `link` on GitHub repository profile
+* link to `"Fork me on GitHub"` at the bottom of document
+* setup presentation title in
+  * `&lt;title&gt;` tag on the top of document in `&lt;head&gt;` tag
```

Comment:
* lista zagłębiona powinna mieć 4 spacje
* punktorem w zagnieżdzonej liście powinien być `+`

Reply:
https://github.com/adam-p/markdown-here/issues 

I guess it's outdated?

Reply:
Czy dodatkowo dodanie pliku eslinta nie ulatwiloby trochę pracy z whitespace'ami?